src/NetscriptFunctions/CodingContract.ts

```diff
@@ -18,11 +19,20 @@ export function NetscriptCodingContract(): InternalAPI<ICodingContract> {
   };
 
   return {
-    attempt: (ctx) => (answer, _filename, _hostname?) => {
+    attempt: (ctx) => (answer: unknown, _filename: unknown, _hostname?: unknown, _type?: unknown) => {
```

Comment:
You don't have to explicitly type these, all parameters are automatically unknown through a process I don't fully understand/recall.

Reply:
Leaving out the explicit types actually results in an error. I'm not sure either, why this is the case. For `getContract`, I can remove them safely though.